src/components/Header/TopNav/LogoLink.js

```diff
@@ -28,16 +28,24 @@ const StyledHomePageLink = styled.div`
 
 const LogoLink = ({
   isHomePage,
+  isEventsPage,
   isServiceOrSpecialityPage,
   fillColorInitial,
   fillColorHover,
   textColor
 }) => {
-  const isSvgLogo = isServiceOrSpecialityPage || isHomePage
```

Comment:
Why not just add `isEventsPage` to this evaluation? It's a lot less disruptive and keeps `HomepageLogo` to only be used once rather than where you're checking for `isEventsPage` below and line 62


Whilst we're here we should rename the HomepageLogo to something else as now its homepage and events logo